src/schemas/org.gnome.shell.extensions.gsnap.gschema.xml

```diff
@@ -129,6 +129,10 @@
 			<default>true</default>
 			<summary>Show tabs for windows in each zone.</summary>
 		</key>
+		<key type = "b" name="use-modifier">
+			<default>false</default>
+			<summary>Use modifier to snap windows</summary>
```

Comment:
The modifier key is not documented. As I see it, we have two ways of fixing it:
- Document that CTRL is the modifier used. At that point the label could be something like: "Hold CTRL to snap windows".
- Make the modifier user-configurable.

Feel free to choose whichever solution do you want to follow as for this PR. As for the user configurable modifier key, it can be also done in a separate PR, so do not feel forced to implement it.

Reply:
I've updated the documentation to `Hold CTRL to snap windows` in the schema and in the prefs_builder.ts file.